packages/ui/components/calendar/src/LionCalendar.js

```diff
@@ -583,6 +584,52 @@ export class LionCalendar extends LocalizeMixin(LitElement) {
     return `${this.msgLit(`lion-calendar:${mode}${type}`)}, ${month} ${year}`;
   }
 
+  /**
+   *
+   * @private
+   */
+  __getEarliestSelectableDate() {
```

Comment:
Can't we merge the `__getEarliestSelectableDate` and `__getLatestSelectableDate` into one function: `__formatDate(date)`?